Formula/python@3.9.rb

```diff
@@ -2,8 +2,8 @@ class PythonAT39 < Formula
   desc "Interpreted, interactive, object-oriented programming language"
   homepage "https://www.python.org/"
   # Keep in sync with python-tk@3.9.
```

Comment:
Please update `python-tk@3.9` formula as well (in a different commit in the same PR)